package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/Kong/kongponents.git"
   },
   "homepage": "https://kongponents.konghq.com/",
-  "main": "./dist/kongponents.umd.js",
+  "main": "./dist/kongponents.cjs",
```

Comment:
https://jsfiddle.net/r672om8t/

issue: The CDN example on the docs site no longer works